nixos/modules/services/networking/wstunnel.nix

```diff
@@ -1,123 +1,152 @@
-{ config, lib, options, pkgs, utils, ... }:
-with lib;
+{ config
+, lib
+, pkgs
+, ...
+}:
+
 let
   cfg = config.services.wstunnel;
-  attrsToArgs = attrs: utils.escapeSystemdExecArgs (
-    mapAttrsToList
-    (name: value: if value == true then "--${name}" else "--${name}=${value}")
-    attrs
+
+  attrsToArgs = attrs: lib.escapeShellArgs (
+    lib.mapAttrsToList
+      (name: value: if value == true then "--${name}" else "--${name}=${value}")
+      attrs
   );
 
-  hostPortToString = { host, port }: "${host}:${builtins.toString port}";
+  hostPortToString = { host, port }: "${host}:${toString port}";
 
   hostPortSubmodule = {
     options = {
-      host = mkOption {
+      host = lib.mkOption {
         description = "The hostname.";
-        type = types.str;
+        type = lib.types.str;
       };
-      port = mkOption {
+      port = lib.mkOption {
         description = "The port.";
-        type = types.port;
+        type = lib.types.port;
       };
     };
   };
 
   commonOptions = {
-    enable = mkOption {
-      description = "Whether to enable this `wstunnel` instance.";
-      type = types.bool;
+    enable = lib.mkEnableOption "this `wstunnel` instance." // {
       default = true;
     };
 
-    package = mkPackageOption pkgs "wstunnel" {};
+    package = lib.mkPackageOption pkgs "wstunnel" { };
 
-    autoStart = mkOption {
-      description = "Whether this tunnel server should be started automatically.";
-      type = types.bool;
-      default = true;
-    };
+    autoStart =
+      lib.mkEnableOption "Whether this tunnel server should be started automatically." // {
```

Comment:
This docstring becomes funny because of the inner working of `mkEnableOption`, [it wraps the given string in `"Whether to enable ${str}."`](https://github.com/NixOS/nixpkgs/blob/a2a0316319cc0625ab75baba5084aacb272ba6d0/lib/options.nix#L98-L105)